src/us/kbase/workspace/test/kbase/HandleAndShockTest.java

```diff
@@ -759,66 +738,67 @@ public void publicWorkspaceHandleTest() throws Exception {
 		String workspace = "publicWS";
 		CLIENT1.createWorkspace(new CreateWorkspaceParams()
 				.withWorkspace(workspace).withGlobalread("r"));
-		Handle h1 = HANDLE_CLIENT.newHandle();
+
```

Comment:
Same questions for this test. A number of assertions were removed, which implies things aren't acting as they're supposed to.

Reply:
👍